bot/api.py

```diff
@@ -16,33 +21,40 @@ def __init__(self, **kwargs):
         else:
             kwargs['headers'] = auth_headers
```

Comment:
Should this be `.update()` instead so that it doesn't override custom headers passed?

Reply:
Isn't that what it's doing in case the headers are given?

Reply:
You're right. I couldn't see the code above.